### ISSUE TITLE
DBI stops transferring files

### ISSUE THREAD

**Issue by rafraleo** [rashevskyv/dbi#325]
Im trying to copy large files to the Saves folder on the switch. I'm doing it by open DBI not in applet mode and pressing X, then pluging the switch on the pc.

But every time I try copying the files, the transfer begins and goes well, but then the loading windows window just closes before the transfer is completed and the file is not fully copied. I tried diffent cable, USB port and even other computer, but couldnt get to copy the files on any.

Am I doing something wrong?

**Comment by rashevskyv** [comment]
What os are you using?

**Comment by bleublurry** [comment]
I have the same issue running on version 658 and windows 11.

I'm using a 3.0 USB-A to USB-C cable connect to a 3.0 USB Hub. The files would transfer for a bit and then shows that the device has disconnected on the PC then automatically reconnects itself prompting the download to stop.

**Comment by zhengyilab** [comment]
Same issue here. 
Tested on DBI v647 & 658, with multiple USB cables, PCs (win10 & win11).AMS 1.6.2 & NS 17.0.1.

Copying serval files (1~4GB in total) to installed game saves, and it randomly stuck on certain files. The PC will beep after waiting a while, and continue to transfer remaining files. This will cause file differences between PC and Switch side, but hardly to check by human.

DBI usually states 'Resizing save ...' or 'Committing FS...' when stuck happens.

Any suggestion would be appreciated!

**Comment by JaySea77** [comment]
Just right click on a folder an select properties. You can see folder size in bytes and amount of files. Then you can see the differences.

**Comment by zhengyilab** [comment]
> Just right click on a folder an select properties. You can see folder size in bytes and amount of files. Then you can see the differences.

Yes, I can tell there're differences on both amount and size, it just hard to find out which file is actually missing or broken... so I can't manually resend that one...
p.s. I'm sending save files of Just Dance 2023, thousands of offline caches with 1~100MB per file...

**Comment by JaySea77** [comment]
That is what I was doing aswell. Took me about 12 hours to get them all transferred successfully.

**Comment by zhengyilab** [comment]
> That is what I was doing aswell. Took me about 12 hours to get them all transferred successfully.

Amazing job as you manually made it! 
I was trying to figure out the reason - just tried format TF card in either exFAT and FAT32, seems no big difference here.
Given the unreliable MTP protocol itself, I think it's normal to break occasionally. However, I believe it's much more unstable when transfer 'saves' compared to install games/copy other files to SD card.

Would like to hear from any advice from DBI owner here...

**Comment by JaySea77** [comment]
The only thing amazing about it is that I never give up. Just remove folders or files and try again. If it is not the same size, do again. Though it was not fun at all.
It tends to break a lot. I seriously hope that it doesn't do that for installs. I guess the problem is that there are many files to transfer in saves and mtp doesn't really play nice with multiple files.
